build/common.props

```diff
@@ -20,4 +20,7 @@
     <PackageReference Include="NETStandard.Library" Version="$(BundledNETStandardPackageVersion)" />
```

Comment:
In most repos, this item group is dead code now. It can be removed now unless there are projects in src that still target .NET Framework